Looks like the travis build failed on unrelated test:
```
======================================================================
91) FAIL: test_retry_still_in_executor (tests.test_jobs.SchedulerJobTest)
----------------------------------------------------------------------
   Traceback (most recent call last):
    tests/test_jobs.py line 3224 in test_retry_still_in_executor
      self.assertEqual(ti.state, State.RUNNING)
   AssertionError: u'success' != u'running'
   - success
   + running
```
I am seeing it pass on the majority of the flavors.

--- a/airflow/jobs.py
+++ b/airflow/jobs.py
@@ -1567,7 +1567,7 @@ class SchedulerJob(BaseJob):
                     "Waiting for processors to finish since we're using sqlite")
                 self.processor_agent.wait_until_finished()
 
-            self.log.info("Harvesting DAG parsing results")
+            self.log.debug("Harvesting DAG parsing results")
             simple_dags = self.processor_agent.harvest_simple_dags()
             self.log.debug("Harvested {} SimpleDAGs".format(len(simple_dags)))
 

